routes/index.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Set up routes for save/publish
+*/
+
+module.exports = function(server) {
+
+  // /projects/* API routes
+  // Handles user-level operations
+  require('./projects/get-projects.js')(server);
+  require('./projects/create-project.js')(server);
+  require('./projects/delete-project.js')(server);
+  require('./projects/publish-project.js')(server);
+
+  // /files/* API routes
+  // Handles project-level operations
+  require('./files/get-project-contents.js')(server);
+  require('./files/get-file-contents.js')(server);
```

Comment:
Exactly, what do you think?  It seems like you have those 3 high level things to worry about, might as well make them all separate routes.